access.c

```diff
@@ -229,22 +229,20 @@ AuthorizeConfig(const connection_t *c)
     {
         if (!o.silent_connection)
         {
-            MessageBoxW(NULL, L"Failed to determine process owner SID", L""PACKAGE_NAME, MB_OK);
+            MessageBoxW(NULL, L"Failed to determine process owner SID", L"" PACKAGE_NAME, MB_OK);
         }
         return FALSE;
     }
     groups = GetProcessTokenGroups();
-    if (IsUserInGroup(sid, groups, admin_group)
-        || IsUserInGroup(sid, groups, o.ovpn_admin_group))
+    if (IsUserInGroup(sid, groups, admin_group) || IsUserInGroup(sid, groups, o.ovpn_admin_group))
     {
```

Comment:
and here it combines args into one line.
There is some logic to it, and I won't fret too much about it.